config/locales/en.yml

```diff
@@ -9,73 +9,76 @@ en:
     create:
       notice: Your Vatican IIIF resource has been successfully created.
       error: There was a problem with adding the Vatican IIIF resource.
+  catalog:
+    parts_default:
+      title: Parts of this manuscript
```

Comment:
@cbeer Can we throw some Google Translate stub for this in it.yml as well?

Reply:
I figured we'd probably take all of our unknown strings and make the Vatican fill it in later?